app/lib/clients/vault/identity_alias.rb

```diff
@@ -0,0 +1,80 @@
+module Clients
+  class Vault
+    module IdentityAlias
+      def put_entity_alias(entity_name, alias_name, auth_method)
```

Comment:
In this file we use "auth_method" and "auth_path" interchangeably.  I think we should just use one.  auth_path is actually more accurate, I think.

Reply:
done